docs/quickstart_insights_core.rst

```diff
@@ -165,6 +165,21 @@ using the following commands::
     [userone@hostone project_dir/insights-core]$ git pull upstream master
     [userone@hostone project_dir/insights-core]$ git push origin master
 
+
+After setup
+==========
```

Comment:
@sourabhtk37 you need to add one more `=` here, the underline has to be the same length as the heading.

Reply:
Fixed. Thanks.